client/app/configuration/configuration.html

```diff
@@ -4,13 +4,28 @@ <h1 class="d-inline">
             <span data-translate="welcome_to_your_manager_1"></span>
             <span data-translate="welcome_to_your_manager_2"></span>
         </h1>
-        <p class="lead"
-           data-ng-bind-html="'core_top_guides_header' | translate: { t0: ConfigurationCtrl.allGuides }"></p>
+        <p
+            class="lead"
```

Comment:
Don't we have a `ui-kit` equivalent ?

Reply:
Not that I know of, and I couldn't find one either